Views/EventBriteField.cshtml

```diff
@@ -0,0 +1,20 @@
+﻿@model Etch.OrchardCore.Fields.EventBrite.ViewModels.DisplayEventBriteFieldViewModel
+@using OrchardCore.Mvc.Utilities;
+
+@{
+    var name = (Model.PartFieldDefinition.PartDefinition.Name + "-" + Model.PartFieldDefinition.Name).HtmlClassify();
+}
+
+@if (Model.HasData)
```

Comment:
Wondering if instead of having everything wrapped within an `if` statement the driver display method should be updated to not return a shape when there is no data.

Reply:
I based this off of `ValuesField.cshtml` and `ResponseiveMediaField.cshtml`, though the former of those two also have different logic to render a message if there is no data.

Reply:
Understood, I think they should probably follow my suggestion as well. Feels cleaner. The only downside is if you wanted to override the template to render something regardless of whether the field had a value, then it wouldn't be possible.